### PR TITLE
Application logs as one view/endpoint

### PR DESCRIPTION
**Pull request by janelletavares**
Fixes https://github.com/meroxa/turbine-project/issues/280

### CODE REVIEW

File: pkg/meroxa/application.go [hunk 1]
```diff
@@ -214,3 +220,23 @@ func (c *client) ListApplications(ctx context.Context) ([]*Application, error) {
 
 	return aa, nil
 }
+
+func (c *client) GetApplicationLogs(ctx context.Context, name string) (*ApplicationLogs, error) {
```
Comment (samirketema) [inline pkg/meroxa/application.go:224]: Thoughts on adding a basic test for this in `pkg/meroxa/application_test.go`?

**Review by samirketema — COMMENTED** [review]
looks good, just a comment on testing

File: pkg/meroxa/application.go (hunk 1 above)
Comment (janelletavares) [reply pkg/meroxa/application.go:224]: ah, yes... that makes sense

**Review by samirketema — APPROVED** [review]
awesome, lgtm! 💯

**Review by jayjayjpg — APPROVED** [review]
This reads great, thank you for the super quick update! :sparkles:

**Review by nassor — APPROVED** [review]

**Pull request merged**